lib/topology_lib_ping/library.py

```diff
@@ -30,7 +30,8 @@
 PING_RE = (
     r'^(?P<transmitted>\d+) packets transmitted, '
     r'(?P<received>\d+) received,'
-    r'( \+(?P<errors>\d+) errors,)? '
+    r'( \+(?P<errors>\d+) errors,)?'
+    r'( \+(?P<duplicates>\d+) duplicates,)? '
```

Comment:
Please check for all possible cases of ping response -
Case 1 (Duplicates and Errors) - 10000 packets transmitted, 10000 received, +19 duplicates, +10 errors, 0% packet loss, time 10035ms
Case 2 (Duplicates only) - 10000 packets transmitted, 10000 received, +19 duplicates, 0% packet loss, time 10035ms
Case 3 (Errors only) - 10000 packets transmitted, 10000 received, +10 errors, 0% packet loss, time 10035ms
Case 4 (No duplicates and no errors) - 10000 packets transmitted, 10000 received, 0% packet loss, time 10035ms